eos_data_distribution/ndn/file.py

```diff
@@ -82,25 +83,32 @@ def mkdir_p(dirname):
 SEGMENT_TABLE_MAGIC = 'EosSgtV1'
 
 # 7 => [0, 0, 0, 0, 0, 1, 1, 1]
+
+
 def num_to_bitmap(n):
     assert 0 <= n <= 255
     return [int(c) for c in list(bin(n)[2:].zfill(8))]
 
 # [0, 0, 0, 0, 0, 1, 1, 1] -> 7
+
+
```

Comment:
Similarly: I think a better fix here would be to put the comment text in a docstring for the function, rather than effectively moving it further away from the function it documents.